pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.isort]
+profile = "black"
+
+[tool.poetry]
+name = "BrainyPedia"
+version = "0.1.0"
+description = "A large scale computational neuroscience knowledge graphs project"
+authors = ["Tek Raj Chhetri <tekraj@mit.edu>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
Is this dependency not covered by the python base image you use in your `FROM` clause in your `Dockerfile`?

Reply:
I am no longer using the poetry because it seems poetry has some issue with docker and when I was testing, I had the similar issues. Will remove this file.